.changeset/warm-meals-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+"effect": minor
```

Comment:
can be patch (and we can target the `main` branch since this is only a signature change)